docs/src/app/components/pages/components/Popover/ExampleAnimation.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Popover from 'material-ui/lib/popover/popover';
+import PopoverAnimationFromTop from 'material-ui/lib/popover/popover-animation-from-top';
+import RaisedButton from 'material-ui/lib/raised-button';
+
+const styles = {
+  popover: {
+    padding: 20,
+  },
+  block: {
```

Comment:
Will do. I was undecided whether to leave it "clean", or to overflow the following example.